frontend/.eslintrc.js

```diff
@@ -70,6 +70,7 @@ module.exports = {
       ],
       2,
     ],
+    "no-multiple-empty-lines": ["error", { max: 1 }],
```

Comment:
Can we please apply the same rule for the API and widget?

Reply:
Yes of course and this rule can resolve the extra line issue
I will include the rule :magic_wand:  and keep you informed